### PR TITLE
Release 6.6.0

### PR DESCRIPTION
**Pull request by acampos1916**
**Features**
#795 #822 SEPA Direct Debit tokenization 
#809 Separate checks for IP and HMAC for notifications received
#812 3DS2/3DS1 fallback
#816 Add the minify exclude element in config 

**Fixes**
#808 Validate checkout agreements before placing order with Google Pay
#813 Remove dependency for Magento setup exception
#814 Fix for Vault with 3DS2 Credit Cards 
#821 3DS storeview URL fix for redirect

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "adyen/module-payment",
   "description": "Official Magento2 Plugin to connect to Payment Service Provider Adyen.",
   "type": "magento2-module",
-  "version": "6.4.0",
+  "version": "6.5.0",
```
Comment (Aleffio) [inline composer.json:5]:
```suggestion
  "version": "6.6.0",
```

**Review by AlexandrosMor — DISMISSED** [review]

**Review by msilvagarcia — DISMISSED** [review]

**Review by AlexandrosMor — APPROVED** [review]

File: Model/Cron.php [hunk 2]
```diff
@@ -585,6 +641,12 @@ protected function _declareVariables($notification)
         $additionalData = !empty($notification->getAdditionalData()) ? $this->serializer->unserialize(
             $notification->getAdditionalData()
         ) : "";
+        if (!empty($additionalData[Vault::RECURRING_DETAIL_REFERENCE])) {
+            $this->_recurringDetailReference = $additionalData[Vault::RECURRING_DETAIL_REFERENCE];
+        }
+        if (!empty($additionalData[Vault::EXPIRY_DATE])) {
+            $this->_expiryDate = $additionalData[Vault::EXPIRY_DATE];
+        }
```
Comment (cyattilakiss) [inline Model/Cron.php:649]: I think it's not used anymore anywhere. Can it be removed?
```suggestion
```

File: Model/Cron.php [hunk 3]
```diff
@@ -239,6 +249,36 @@ class Cron
...
+    /**
+     * @var
+     */
+    protected $_recurringDetailReference;
+
+    /**
+     * @var
+     */
+    protected $_expiryDate;
+
```
Comment (cyattilakiss) [inline Model/Cron.php:261]: same here
```suggestion
```

File: Model/Cron.php (hunk 2 above)
Comment (acampos1916) [reply Model/Cron.php]: Indeed it can be removed in both locations

File: Model/Cron.php (hunk 3 above)
Comment (acampos1916) [reply Model/Cron.php]: Indeed it can be removed in both locations

**Review by cyattilakiss — APPROVED** [review]

**Review by msilvagarcia — APPROVED** [review]

**Pull request merged**